src/ImpurityDensities/PtypePNJunctionImpurityDensity.jl

```diff
@@ -16,19 +16,21 @@ function PtypePNJunctionImpurityDensity{T}(
     lithium_annealing_temperature::T,
     lithium_annealing_time::T,
     contact_with_lithium_doped::G,
+    inactive_contact_id::Int,
     bulk_imp_model::AbstractImpurityDensity{T},
     distance_to_contact::Function = pt::AbstractCoordinatePoint{T} -> ConstructiveSolidGeometry.distance_to_surface(pt, contact_with_lithium_doped)
     ) where {T <: SSDFloat, G <: Union{<:AbstractGeometry, Nothing}}
-    surface_imp_model = ThermalDiffusionLithiumDensity{T}(lithium_annealing_temperature, lithium_annealing_time, contact_with_lithium_doped, distance_to_contact=distance_to_contact)
+    surface_imp_model = ThermalDiffusionLithiumDensity{T}(lithium_annealing_temperature, lithium_annealing_time, contact_with_lithium_doped, inactive_contact_id, distance_to_contact=distance_to_contact)
     PtypePNJunctionImpurityDensity{T}(surface_imp_model, bulk_imp_model)
 end
 
 function ImpurityDensity(T::DataType, t::Val{:PtypePNjunction}, dict::AbstractDict, input_units::NamedTuple)
     lithium_annealing_temperature = _parse_value(T, get(dict, "lithium_annealing_temperature", 623u"K"), input_units.temperature)
     lithium_annealing_time = _parse_value(T, get(dict, "lithium_annealing_time", 18u"minute"), internal_time_unit)
     contact_with_lithium_doped = get(dict, "contact_with_lithium_doped", nothing)
+    inactive_contact_id = get(dict, "doped_contact_id", 1)
```

Comment:
Is `1` a good default? Maybe we want to use `0` or `-1`, because they can never me a valid contact ID and we can catch this later. If someone were to spell `doped_contact_id` wrong, they might get unwanted behavior.